.env.blockscoutstagingrc1

```diff
@@ -0,0 +1,519 @@
+# Don't use "//" for comments in this file.
+# This file is meant to be executed as a bash script for testing.
+ENV_TYPE="production"
+
+GETH_VERBOSITY=2
+GETH_ENABLE_METRICS=false
+
+VM_BASED=true
```

Comment:
@jcortejoso I saw a mainnet archive node pod, should we change to false? Does it even have any effect now?

Reply:
Hi @carterqw2. It should only has impact on the `testnet` package when `BLOCKSCOUT_OVERRIDE_*_ENDPOINT` are set like here.